dev/com.ibm.ws.classloading_sharedclass_fat/fat/src/io/openliberty/classloading/sharedclasses/fat/FATSuite.java

```diff
@@ -0,0 +1,164 @@
+                                                                   io.openliberty.classloading.sharedclasses.warlib.b.B.class.getPackage().getName());
+            SHARED_CLASSES_EAR_LIB = ShrinkHelper.buildJavaArchive(SHARED_CLASSES_EAR_LIB_NAME + ".jar", //
+                                                                   io.openliberty.classloading.sharedclasses.earlib.a.A.class.getPackage().getName(), //
+                                                                   io.openliberty.classloading.sharedclasses.earlib.b.B.class.getPackage().getName());
+            SHARED_CLASSES_SERVER_LIB = ShrinkHelper.buildJavaArchive(SHARED_CLASSES_SERVER_LIB_NAME + ".jar", //
+                                                                    io.openliberty.classloading.sharedclasses.serverlib.a.A.class.getPackage().getName(), //
+                                                                    io.openliberty.classloading.sharedclasses.serverlib.b.B.class.getPackage().getName());
+
+            SHARED_CLASSES_EJB = ShrinkHelper.buildJavaArchive(SHARED_CLASSES_EJB_NAME + ".jar", //
+                                                               io.openliberty.classloading.sharedclasses.ejb.TestEJB.class.getPackage().getName(), //
+                                                               io.openliberty.classloading.sharedclasses.ejb.a.A.class.getPackage().getName(), //
+                                                               io.openliberty.classloading.sharedclasses.ejb.b.B.class.getPackage().getName());
+
+            SHARED_CLASSES_WAR = ShrinkHelper.buildDefaultApp(SHARED_CLASSES_WAR_NAME + ".war",
+                                                              io.openliberty.classloading.sharedclasses.war.TestSharedClassesWar.class.getPackage().getName(), //
+                                                              io.openliberty.classloading.sharedclasses.war.b.B.class.getPackage().getName(), //
+                                                              io.openliberty.classloading.sharedclasses.war.a.A.class.getPackage().getName())
+                            .addAsLibrary(SHARED_CLASSES_WAR_LIB);
+
+
+            SHARED_CLASSES_RESOURCE_ADAPTOR = ShrinkHelper.buildJavaArchive(SHARED_CLASSES_RESOURCE_ADAPTOR_NAME + ".jar", //
+                                                                            io.openliberty.classloading.sharedclasses.resourceadaptor.TestResourceAdapter.class.getPackage().getName(), //
+                                                                            io.openliberty.classloading.sharedclasses.resourceadaptor.a.A.class.getPackage().getName(), //
+                                                                            io.openliberty.classloading.sharedclasses.resourceadaptor.b.B.class.getPackage().getName());
+
+            // Using JavaArchive to create a RAR here because the ResourceAdapterArchive does allow packages to be added directly
```

Comment:
```suggestion
            // Using JavaArchive to create a RAR here because the ResourceAdapterArchive does not allow packages to be added directly
```
I believe you were missing the `not`, but you can correct me if I am wrong.